docs/porting/libraries.md

```diff
@@ -1,5 +1,311 @@
-# 🔧 Porting your libraries
+# Porting Libraries to .NET Core
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+By [Phillip Carter](https://github.com/cartermp)
+
+Libraries are some of the most natural projects to port from .NET Framework to .NET Core.  This article covers all of the steps you'll need to take to get to the point where you can focus on your code.  It covers the following:
+
+- Learning about important discontinued technologies for .NET Core
+- Picking the correct .NET Platform Standard for your library
+- Understanding the basics of the .NET Core project model
+- Understanding how multitargeting works in .NET Core
+- If applicable, retargeting your code to .NET Framework 4.6.1
+- Determining the portability of your code
+- Picking the correct project system in Visual Studio based on your needs
+- Porting your tests
+- Recommended approach for porting your code
+
+It's quite a lot of content, but porting is a task that will take some time, especially if you have a large codebase.  You should also be prepared to adapt the guidance here as needed to best fit your code.  Every codebase is different, so this article attempts to frame things in a flexible way, but you may find yourself needing to diverge from the prescribed guidance.
+
+## Prerequisites
+
+This article assumes you are using Visual Studio 2015 on Windows.
```

Comment:
Say a word about why, and what possible workarounds could exist now or in the future if that's not an option.